tests/pass/fs.rs

```diff
@@ -364,6 +365,27 @@ fn test_rename() {
     remove_file(&path2).unwrap();
 }
 
+fn test_canonicalize() {
+    use std::fs::canonicalize;
+    let dir_path = prepare_dir("miri_test_fs_dir");
+    create_dir(&dir_path).unwrap();
+    let path = dir_path.join("test_file");
+    drop(File::create(&path).unwrap());
+
+    let p = canonicalize(format!("{}/./test_file", dir_path.to_string_lossy())).unwrap();
+    assert_eq!(p.to_string_lossy().find('.'), None);
+
+    remove_dir_all(&dir_path).unwrap();
+
+    // Make sure we get an error for long paths.
+    use std::convert::TryInto;
+    #[cfg(not(windows))]
```

Comment:
This entire file does not run on windows so this `cfg` is redundant.

Reply:
Without this CI failed. I will double check though.